v2/analyzer-deployment.yaml

```diff
@@ -23,12 +23,12 @@ spec:
             cpu: 100m
             memory: 100Mi
         env:
-        #- name: VCAP_SERVICES_TONE_ANALYZER_0_CREDENTIALS_USERNAME
-        #  value: ...
-        #- name: VCAP_SERVICES_TONE_ANALYZER_0_CREDENTIALS_PASSWORD
-        #  value: ...
-        - name: USE_HTTPS
-          value: True
+        #- name: "VCAP_SERVICES_TONE_ANALYZER_0_CREDENTIALS_USERNAME"
+        #  value: "..."
+        #- name: "VCAP_SERVICES_TONE_ANALYZER_0_CREDENTIALS_PASSWORD"
+        #  value: "..."
+        - name: "USE_HTTPS"
```

Comment:
I guess I can understand why quotes might be needed around the value, but why the name? If you look at the normal Kube docs (e.g. https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/ ) they don't use quotes on "name".  Is this an Istio bug?